### PR TITLE
[READY] Adding Coffee tree to locations

### PR DESCRIPTION
**Pull request by Nymall**
Ads Coffee Tree to the following locations:

* Botanical Gardens(one tree with sign)
* Coffee Shop 2(replacing two of the corner bushes)

And a new OM Plot:

* ~~Coffee Orchard(20 trees, small coffee shop towards road, storage in back)~~

Any other potential plots? If so, where would you like to see it?

### CODE REVIEW

File: data/json/mapgen/cosmics_additions/GardenBotanical.json [hunk 1]
```diff
@@ -196,7 +197,8 @@
         { "signage": "CATTAIL", "x": 4, "y": 14 },
         { "signage": "DATURA", "x": 4, "y": 19 },
         { "signage": "FUNGUS", "x": 9, "y": 14 },
-        { "signage": "CACTUS", "x": 9, "y": 19 }
+        { "signage": "CACTUS", "x": 9, "y": 19 },
+        { "signage": "KENTUCKY COFFEE", "x": 15, "y": 3 }
```
Comment (cosmiccoincidence) [inline data/json/mapgen/cosmics_additions/GardenBotanical.json]: I'd suggest making the sign just read "COFFEE" as to fit with the rest of the signs being all one word (APPLE, PINE, etc.).

Comment (cosmiccoincidence) [reply data/json/mapgen/cosmics_additions/GardenBotanical.json]: Ok, fair enough. I didn't know it was difference and had to look it up to find out its a completely different species.

**Comment by Leland** [comment]
Are there actually coffee orchards in New England?

**Comment by cake-pie** [comment]
> Are there actually coffee orchards in New England?

[Wikipedia](https://en.wikipedia.org/wiki/Coffee_production_in_Hawaii):
> One of only two states in the United States of America able to grow coffee plants commercially is Hawaii, the other being California. However, it is not the only coffee grown on U.S. soil; for example, Puerto Rico has had a coffee industry for some time, although it is not a state but a U.S. territory.

**Comment by cosmiccoincidence** [comment]
> Kentucky coffee

> it's a completely different species.

https://en.m.wikipedia.org/wiki/Kentucky_coffeetree

> Are there actually coffee orchards in New England?

There are no coffee orchards anywhere near New England, and googling about the the Kentucky coffee tree I don't think many people actually take this species serious enough to grow it in orchards. It seems some botanical nurseries may have small ones to sell for people who want to plant one at their home or in a park though.

**Comment by Nymall** [comment]
Very well, I'll mark this one as done then. We have two more locations where it is used. I'm kinda swamped with work right now so I don't have a lot of dev time, but might try and think up some more possible locations in the future.

**Comment by vlad1492** [comment]
A tasty beverage, but not actually coffee. No caffeine.

**Comment by DracoGriffin** [comment]
Yeah, not sure how [realistic or feasible Kentucky coffee tree](https://archive.org/stream/biostor-159967/biostor-159967_djvu.txt) is as:
>No one claimed to enjoy the taste or the experience. All agreed that an overpowering and persistent af- 
tertaste remained.

>The second author of the present paper drank about 1/3 cup of sweetened KCT “coffee” every day for 2 weeks, hoping that, with familiarity, the flavor might improve for him; it did not. 

>Yet, in contrast, some individuals rate the beverage more highly. Brill (2002) called it “the world’s best caffeine-free coffee substitute”;

>Later, Rogers (1917) wrote: “How the pioneers ever crushed [the seeds] is a puzzle to all who have tried to break one with a nutcracker.”

>The next step was to establish a method of breaking the seeds, which were still rock hard. Without a corn grinder available (and fear that if we did have one, the seeds would break the grinder), we used pliers and a mortar and pestle. The seeds were first cracked open one-by-one with the pliers over the mortar. 
As they were cracked, most shattered or exploded so the cracking was done under a cloth that covered the mortar.

>Nevertheless we do not recommend the “coffee” until more is known about the poisonous compounds in the seeds.

>Other than the reports of Brill (2002), Brill & Dean (1994), Phillips (1998), and Serlin (1977) and the recent brewing of “coffee” from KCT seeds here at Northern Kentucky University, we have located no evidence of significant extant use of the beverage. Even if current edible-plant books mention the use, we suggest that KCT “coffee” is mostly a curiosity from a time now past

**Comment by vlad1492** [comment]
I wouldn't know how to run a gas chromatograph analysis even if I had the gear. But Mr. Internet sat 'no caffeine' : 

https://permies.com/t/6033/Opinions-Kentucky-Coffee-Tree-taste
 "The pod of the Kentucky coffee tree makes the world's best tasting caffeine-free coffee"

http://www.naturalmedicinalherbs.net/herbs/g/gymnocladus-dioica=kentucky-coffee-tree.php
"The roasted seed is a caffeine-free coffee substitute. "

**Comment by cosmiccoincidence** [comment]
Indeed I got the same results when googling.

**Pull request merged**